airbyte-integrations/connectors/source-mailgun/source_mailgun/manifest.yaml

```diff
@@ -52,7 +52,7 @@ definitions:
     cursor_granularity: "PT0.000001S"
     lookback_window: "P31D"
     start_datetime:
-      datetime: "{{ config['start_date'] }}"
+      datetime: "{{ config['start_date'] if config['start_date'] else day_delta(-3, format='%Y-%m-%dT%H:%M:%SZ') }}"
```

Comment:
I think it's better to define a longer default start date, like 90 days.
You can improve the expression to:
`"{{ config.get('start_date', day_delta(-3, format='%Y-%m-%dT%H:%M:%SZ')) }}"`

Please bum the connector version (dockerfile, metadata and doc changelog).
Also add a comment in the documentation explaining the default date.